tasks/sealights/sealights-go-instrumentation/0.1/README.md

```diff
@@ -0,0 +1,3 @@
+# Tekton Task: Sealights Go Instrumentation
+
+Please read/update [test metadata docs](../../../../docs/qe-available-tasks/sealights/go-instrumentation/go-instrumentation.md) from docs folder.
```

Comment:
would it make sense to start documenting tasks/stepactions in build-definitions style? i.e. README.md in every version, which is useful when something changes in the task and the documentation needs to change.

example: https://github.com/konflux-ci/build-definitions/tree/main/task/sast-snyk-check/0.3

Reply:
I think yes. Also I think if we use gitbook the .md files should be under docs foldee